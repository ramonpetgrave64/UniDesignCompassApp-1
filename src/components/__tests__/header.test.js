import React from "react";
// import renderer from "react-test-renderer";
// import {render, fireEvent, wait} from 'react-testing-library';
import {configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import {mount, shallow, render} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import Header from "../header";
import 'react-testing-library/cleanup-after-each';

configure({adapter:new Adapter()});
const tree = shallow(<Header siteTitle="Universal Compass Design"/>);

describe("Header", () => {
  it("Check Render",()=>{
      expect(shallowToJson(tree)).toMatchSnapshot();
  });
  it("Check Constructor Function",()=>{
    let func=jest.spyOn(tree.instance(),"constructor")
    expect(func).toHaveBeenCalledTimes(0);
})
});
