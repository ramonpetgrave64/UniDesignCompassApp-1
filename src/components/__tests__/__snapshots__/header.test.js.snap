// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Check Render 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
